index.html

```diff
@@ -8,7 +8,71 @@
     <link href="css/style.css" type="text/css" rel="stylesheet" />
   </head>
   <body>
-    <img src="img/fish1.jpg" title="First fish">
-    <img src="img/fish2.ppg" title="Second fish">
+
+    <img id="headerfish" src="img/fish 1.jpg" title="First fish" />
+    <h1>Welcome to Scott's Cookie Store!</h1>
+    <h2 id= "Slogan">The store where apperently we thought it was a good idea to put fish in the cookies</h2>
+    <p>Here at Scott's fish store, we sell a large variety of fish cookies to our dedicated fan base all around the seagull population and now we are bringing our amazing fish cookies to a store near you!</p>
+    <P>Dont let us speak for our cookies, let our cookies do the talking! We make our cookies out of 70% real fish! thats 10% more than what you get out of the other fish cookie stores!</P>
+    <section>
+      <img id="sellout" src="img/fish3.jpg" title="third fish" />
+      <img id="sellout" src="img/frostedcookie.jpg" title="cookie that is froosted" />
```

Comment:
You should never have more than one copy of a specific ID on the page, they are meant to be unique. If you want to mark multiple things with the same label, use a `class` instead.